doc/kubernetes/modules/ROOT/partials/aurora/aurora-create-peering-connections.adoc

```diff
@@ -0,0 +1,214 @@
+. Retrieve the Aurora VPC
++
+.Command:
+[source,bash]
+----
+aws ec2 describe-vpcs \
+  --filters "Name=tag:AuroraCluster,Values=keycloak-aurora" \
+  --query 'Vpcs[*].VpcId' \
+  --region eu-west-1 \
+  --output text
+----
++
+.Output:
+[source,json]
+----
+vpc-0b40bd7c59dbe4277
+----
++
+. Retrieve the ROSA cluster VPC
+.. Login to the ROSA cluster using `oc`
+.. Retrieve the ROSA VPC
++
+.Command:
+[source,bash]
+----
+NODE=$(oc get nodes --selector=node-role.kubernetes.io/worker -o jsonpath='{.items[0].metadata.name}')
+aws ec2 describe-instances \
+  --filters "Name=private-dns-name,Values=${NODE}" \
+  --query 'Reservations[0].Instances[0].VpcId' \
+  --region eu-west-1 \
+  --output text
+----
++
+.Output:
+[source,json]
+----
+vpc-0b721449398429559
+----
++
+. Create Peering Connection
++
+.Command:
+[source,bash]
+----
+aws ec2 create-vpc-peering-connection \
+  --vpc-id vpc-0b721449398429559 \# <1>
```

Comment:
These clues are explained below and they are numbered, which creates a nested list which again seems little bit off. If someone knows, how to do it better, that would be great.

Reply:
Disregard this comment. In the compiled version it looks good!